### PR TITLE
fix: [PL-27002]: UI changes for Phishing Bug

### PR DESCRIPTION
**Pull request by sumanmabd**
### Summary

In the notify button of the Pipelines, When we try to create the notification using email, Removed the Subject and Body Section. Below are the before and after screenshots attached respectively.

#### Screenshots

Screenshot 2022-07-25 at 5 34 28 PM

Screenshot 2022-07-25 at 12 59 44 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/15-notifications/modals/ConfigureNotificationsModal/views/ConfigureEmailNotifications/ConfigureEmailNotifications.tsx [hunk 1]
```diff
@@ -117,9 +109,7 @@ export const TestEmailNotifications: React.FC<{ onClick?: () => void; buttonProp
         accountId,
         type: 'EMAIL',
         recipient: testData.to,
-        notificationId: 'asd',
-        subject: testData.subject,
-        body: testData.body
+        notificationId: 'asd'
```
Comment (depanshu-dhiman) [inline src/modules/15-notifications/modals/ConfigureNotificationsModal/views/ConfigureEmailNotifications/ConfigureEmailNotifications.tsx:112]: Ignoring as this was there before as well.

**Review by depanshu-dhiman — DISMISSED** [review]

**Comment by sumanmabd** [comment]
dockerize

**Comment by sumanmabd** [comment]
dockerize <notifytest>

**Comment by sumanmabd** [comment]
dockerize PL-27002

**Comment by sumanmabd** [comment]
trigger cypress

**Comment by depanshu-dhiman** [comment]
trigger cypress

**Comment by sumanmabd** [comment]
trigger cypress

**Comment by sumanmabd** [comment]
trigger cypress

**Review by depanshu-dhiman — APPROVED** [review]

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Review by upadhyay-an — APPROVED** [review]

**Pull request merged**